dev/com.ibm.ws.kernel.service/src/com/ibm/ws/kernel/service/location/internal/WsLocationAdminImpl.java

```diff
@@ -227,6 +231,7 @@ public static WsLocationAdminImpl getInstance() {
      *                                      if bootstrap library location or instance root don't exist.
      */
     protected WsLocationAdminImpl(Map<String, Object> config) {
+        BundleContext bundleContext = FrameworkUtil.getBundle(getClass()).getBundleContext();
```

Comment:
I would prefer this get passed in as an argument to the constructor.  Then for `com.ibm.ws.kernel.service.location.internal.WsLocationAdminImpl.createLocations(Map<String, Object>)` you would pass in `null`.   The code should handle when there is no BundleContext available.